web/testing/promise-api/index.html

```diff
@@ -36,9 +36,24 @@
 
     <!-- Initialization: set paths to keyboards, resources and fonts as required -->
     <script>
+      // Thank you to https://stackoverflow.com/questions/22607150/getting-the-url-parameters-inside-the-html-page.
+      var GetURLParameter = function(sParam) {
```

Comment:
```suggestion
      var getUrlParameter = function(sParam) {
```

Reply:
But, please use [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams); compat: `https://caniuse.com/?search=URLSearchParams`

Reply:
Done, but the "booleans" returned by getUrlParameter still come back `true` and `false` 😞

Reply:
nvm, fixed the bug so it returns a boolean now